### ISSUE TITLE
--retrieval_results

### ISSUE THREAD

**Issue by rosarubus72** [Go2Heart/EchoSight#8]
--retrieval_results: The path of the retrieval results file. Where can the retrieval results file be obtained?

**Comment by Go2Heart** [comment]
see scripts/test_reranker.sh's `save_result_path`

**Comment by rosarubus72** [comment]
Thanks for reply！

**Comment by rosarubus72** [comment]
我在跑test_reranker.py中的perform_vqa精度的时候遇到报错：
tensorflow/core/framework/op_kernel.cc:1839] OP_REQUIRES failed at lookup_table_init_op.cc:152 : FAILED_PRECONDITION: Error executing an HTTP request: libcurl code 77 meaning 'Problem with the SSL CA cert (path? access rights?)', error details: error setting certificate verify locations:  CAfile: /etc/ssl/certs/ca-certificates.crt CApath: none
         when reading gs://cloud-tpu-checkpoints/bert/keras_bert/uncased_L-12_H-768_A-12/vocab.txt
         经过查找报错原因，可能是需要注册Google Cloud Platform？

**Comment by Go2Heart** [comment]
该BEM评测代码使用了E-VQA的官方评测代码([link](https://github.com/google-research/google-research/blob/master/encyclopedic_vqa/evaluation_utils.py))，您遇到的问题可能是网络问题，请确认相关网络访问正常后继续使用。

**Comment by rosarubus72** [comment]
好的，感谢！

**Comment by rosarubus72** [comment]
InfoSeek 数据集采用 VQA 精度和 Relaxed Accuracy,请问这个精度是通过test_reranker.py跑出来了的吗，这个精度对应的是eval score吗？

**Comment by Go2Heart** [comment]
不是，是得到了result之后使用infoseek的官方脚本进行测试的。

**Comment by rosarubus72** [comment]
好的，感谢！
